tests/unit_tests/db/test_db_handler.py

```diff
@@ -613,6 +622,12 @@ def test_get_collections(db, db_config):
     collections_from_name = db.get_collections(db_config["db_simulation_model"])
     assert isinstance(collections_from_name, list)
     assert "telescopes" in collections_from_name
+    assert "fs.files" in collections_from_name
+
+    collections_no_model = db.get_collections(db_config["db_simulation_model"], True)
+    assert isinstance(collections_no_model, list)
+    assert "telescopes" in collections_no_model
+    assert "fs.files" not in collections_no_model
```

Comment:
test also for metadata

Reply:
I consciously didn't do that, as the open PR #1123 is removing the metadata collection. But as we might want to use old DBs, it maybe good to test it. Therefore followed your suggestion.